### DISCUSSION TITLE
Inference techniques

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#15] [Q&A] [answered]
What is resolution technique?Share ur point of views on how it helps in proving goal facts

**Comment by GouniSahithi** [comment, marked as answer]
Resolution technique is used primarily in automated reasoning and logic.
Resolution operates within propositional logic and first- order logic.It converts statements into a standard form to facilitate the resolution process.
HELPS IN PROVING GOAL FACTS:
1.Efficiency:The technique can be very efficient for certain types of problems.
It helps to identify the validity of complex statement through a structural process
2.Clarity:By breaking down arguments into clauses, resolution provides a clear framework for reasoning.
3.structure:it helps in identifying relationships between different propositions and understanding the logic
4.Applications : resolution is applied in various domains.
